ui/app/templates/components/masked-input.hbs

```diff
@@ -18,7 +18,7 @@
   {{#if allowCopy}}
     <CopyButton
       @clipboardText={{value}}
-      @class="copy-button button is-compact"
```

Comment:
Removing this makes things look mis-aligned in read-only mode (sorry there's a lot going on in this component).

![screen shot 2018-12-06 at 11 21 33 am](https://user-images.githubusercontent.com/39469/49601466-83914800-f94b-11e8-9226-befea910e342.png)